src/metatensor/models/cli/conf/base.yaml

```diff
@@ -0,0 +1,3 @@
+device: "cpu"
```

Comment:
Perfect, I'll make sure the SOAP-BPNN works on GPU (after the spherical expansion) in a future PR